File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/mad/Java/Go/s287007304_comments_removed.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	line, err := reader.ReadString('
')
	if err != nil {
		fmt.Println("Error reading input:", err)
		return
	}
	line = strings.TrimSpace(line)
	s := strings.Split(line, " ")

	if len(s) != 2 {
		fmt.Println("Invalid input: expected two integers")
		return
	}

	a, err := strconv.Atoi(s[0])
	if err != nil {
		fmt.Println("Error parsing first integer:", err)
		return
	}

	b, err := strconv.Atoi(s[1])
	if err != nil {
		fmt.Println("Error parsing second integer:", err)
		return
	}

	if b%a == 0 {
		fmt.Println(a + b)
	} else {
		fmt.Println(b - a)
	}
}

// End of Code

File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/mad/Java/C++/s287007304_comments_removed.cpp
#include <iostream>
#include <sstream>
#include <string>

std::string readLine() {
    std::string line;
    if (std::getline(std::cin, line)) {
        return line;
    } else {
        return "Error reading line";
    }
}

int main() {
    std::string input = readLine();
    std::istringstream iss(input);
    int a, b;
    
    if (!(iss >> a >> b)) {
        std::cerr << "Error parsing integers" << std::endl;
        return 1;
    }

    if (b % a == 0) {
        std::cout << a + b << std::endl;
    } else {
        std::cout << b - a << std::endl;
    }

    return 0;
}
// End of Code
